balrogscript/src/balrogscript/submitter/cli.py

```diff
@@ -261,6 +306,81 @@ def update_latest():
 
         retry(update_latest, sleeptime=2, max_sleeptime=2, attempts=10)
 
+    def run_backend2(
+        self, platform, buildID, productName, branch, appVersion, locale, hashFunction, extVersion, schemaVersion, isOSUpdate=None, **updateKwargs
+    ):
+        log.info("Using backend version 2...")
+        requests_api = get_balrog_api(auth0_secrets=self.auth0_secrets)
+        headers = {"Accept-Encoding": "application/json", "Accept": "application/json", "Content-Type": "application/json", "Referer": self.api_root}
+
+        def do_request(url, method="get", data=None):
+            logging.debug("Balrog request to %s via %s", url, method.upper())
+            logging.debug("Data sent: %s", None)
+            before = time.time()
```

Comment:
You don't need to do this by hand. You can look at `resp.elapsed.total_seconds()` to get this.

Reply:
Followup fodder? Since this is how the original v1 api does it?

Reply:
:man_shrugging: Your call